modules/ServerAPI/src/client/Thing.coffee

```diff
@@ -187,6 +189,17 @@ class Thing extends Backbone.Model
 			if !data.has ('recordedDate') || data.get('recordedDate') == null
 				data.set recordedDate: rDate
 
+	setName: =>
```

Comment:
This name is not saved in the database but rather is used for UI components (such as the new ACASThingBrowser) to be able to provide a default display name for the thing type/kind.

Reply:
I think `name` is too vague for this. Maybe "Class Name" or "Thing Kind Display Name"? I originally thought this meant a preferred label, and I could see this being confusing as many Things have "names" that are particular to an instance rather than the name of the whole class of Things.